terraform/modules/api/main.tf

```diff
@@ -69,6 +69,10 @@ resource "aws_api_gateway_rest_api" "scale" {
   name        = "SCALE:EU2:${upper(var.environment)}:API:Shared"
   description = "SCALE API Gateway"
 
+  endpoint_configuration {
```

Comment:
This looks like a new addition (compared to the pre-change version) - but guess it has no real impact - just checking

Reply:
Can remove it but makes the type explicit which seems like a sensible thing given the circumstances.. don't mind 🤔